daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/dfa/TextDelimitedUnparser.scala

```diff
@@ -248,6 +248,8 @@ class TextDelimitedUnparser(override val context: TermRuntimeData)
     escapeEscapeChar: MaybeChar, state: UState): (String, Boolean) = {
     Assert.invariant(delims != null)
     Assert.invariant(field != null)
+    if (hasEscCharAsDelimiter)
+      UnparseError(One(context.schemaFileLocation), One(state.currentLocation), "Delimiters may not begin with the dfdl:escapeCharacter (%s)", escapeChar)
```

Comment:
If every schema and every TDML test fails the checkDelimiterEscapeConflict check before they can hit this line, it may be difficult to write a test that covers this line.  We'd have to write a unit test that calls this specific method to hit this line or tell Codecov to ignore this line.

Reply:
I was able to write a test that covers this without needing to do a unit test.  Just needed a test where the separator is determined at runtime.